.github/workflows/test-and-binaries.yml

```diff
@@ -0,0 +1,279 @@
+name: Deployment
+
+on: [push]
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v1
+        with:
+          depth: 1
+          submodules: false
+      - uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: |
+            ${{ runner.OS }}-gradle-${{ env.cache-name }}-
+            ${{ runner.OS }}-gradle-
+            ${{ runner.OS }}-
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 13
+      - name: Build classes
+        run: ./gradlew -Pdev=true classes
+      - name: Upload build
```

Comment:
This does not seem to work, as the following tests seem to still re-build from the source. I also don't understand why uploading 40mb takes 15min...